README.md

```diff
@@ -18,6 +18,10 @@ Because I want an easy way to see where my disk is being used.
 
 * `cargo install du-dust`
 
+#### Brew (Mac OS)
```

Comment:
Maybe we could add a 🍺-Emoji 🙂

Reply:
Almost necessary, indeed ;)

Reply:
😁🍻 Since it is a very small change, it would be good to squash them into one commit and do a force-push 🚀